src/data/gqlQueries.js

```diff
@@ -114,9 +117,10 @@ export function createPost(post) {
       createPost(input: {
         orgID: "${post.orgID}",
         type: ${post.type},
-        title: "${post.title}",
-        description: "${post.description}",
-        destination: "${post.destination}",
+        title: """${post.title}""",
+        description: """${post.description}""",
```

Comment:
what's the `"""` about again?

Reply:
That prevents quotes and newlines from breaking the GQL syntax.